CONTRIBUTING.md

```diff
@@ -0,0 +1,133 @@
+# Contribution guidelines
+
++ [Before adding or editing a recipe](#before-adding-or-editing-a-recipe)
+  + [Labels](#labels)
++ [Recipe Structure](#recipe-structure)
+  + [Badges](#badges)
++ [Pull requests](pull-requests)
++ [Instructions for maintainers](#instructions-for-maintainers)
+
+## Before adding or editing a recipe
+
+ 1. Search this repo's [issues](https://www.github.com/foundersandcoders/nazareth-recipes/issues) to see if an issue exists for the recipe.
+ 2. If the issue does not exist, create it.
+ 3. If your new issue relates to any others, reference those issues in the body. This enables others to follow the history of the topic.
+ 4. Assign [the appropriate label](#labels). **All recipe issues must have labels**
+ 5. Once you are sure of what you need to do and that it is needed, assign yourself to the issue.
+ 6. Clone, and create a new branch for your work
+
+### Labels
+Advice for labeling your recipes:
+
+  - Indicate what meal your recipe might apply to (breakfast, lunch or dinner) - it can apply to more than one.
+  - Indicate whether or not the meal is vegetarian friendly
+  - Add any other labels you feel might be appropriate.
+
+## Recipe Structure
+Use the following skeleton guide
```

Comment:
I can't approve until this has been discussed- I need reassurance that PR's that don't exactly match this structure won't be automatically discarded.